pkg/apiserver/profiling/router.go

```diff
@@ -303,6 +304,15 @@ func (s *Service) viewSingle(c *gin.Context) {
 		_ = c.Error(err)
 		return
 	}
+	if outputType == "graph" {
+		svgContent, err := convertProtobufToSVG(content, task)
+		if err != nil {
+			_ = c.Error(err)
+			return
+		}
+		content = svgContent
+	}
+
 	c.Data(http.StatusOK, "image/svg+xml", content)
```

Comment:
Seems that previously the content type is set wrongly. When proto is served, its content type should not be image/svg+xml.